layers-bsp.csv

```diff
@@ -1,15 +1,18 @@
 Relative GIT Repository,Layer subdirectory in repository,Tag/Branch to checkout in develop MSC-LDK branch
-thirdparty/yocto,meta,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-freescale,,hardknott-5.10.35-2.0.0
-thirdparty/meta-freescale-distro,,hardknott-5.10.35-2.0.0
-thirdparty/meta-imx,meta-bsp,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-imx,meta-sdk,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-qt5,,hardknott-imx-5.10.35-2.0.0
-thirdparty/meta-nxp-demo-experience,,hardknott-5.10.35-2.0.0
-msc_ol99/meta-msc-arm-extensions,,
-msc_ol99/meta-msc-ldk-marvell,,
-thirdparty/meta-openembedded,meta-oe,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-networking,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-python,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-multimedia,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-gnome,hardknott-imx-5.10.35-2.0.0-msc
+meta-processor-sdk,,dunfell-08.04.01.04-msc
+meta-arago,meta-arago-distro,dunfell-08.04.01.04-msc
+meta-arago,meta-arago-extras,dunfell-08.04.01.04-msc
+meta-qt5,,dunfell-08.04.01.04-msc
+meta-virtualization,,dunfell-08.04.01.04-msc
+meta-ti,,dunfell-08.04.01.04-msc
```

Comment:
The suffix -msc is only necessary when there are changes in the repository by MSC/Avnet or when a necessary branch did not yet exist.
We prefer not to do any changes in common layers.

Reply:
ok. Understood now.
I don't know about naming convetion. So I have copied from imx and created <YOCTO_BRANCH-TI_VERSION-msc>

We haven't made any changes in common repos apart from "zsh" package patches which you have applied on the common yocto repo meta-openembedded
https://github.com/anilpatelavnet/meta-openembedded/commit/a77cd97ad8f95b251ce535733ad6eb0dc88d2b1e
https://github.com/anilpatelavnet/meta-openembedded/commit/b2e33a546843748b0ade82d267dc3abd3edd2d9d